client/src/components/atoms/CommentEditorButton.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <q-btn
+    :aria-label="$t(props.ariaLabel)"
```

Comment:
So, anything that's not a prop will automatically be applied to the root element as an attr via inheritAttrs.  I noticed you're specifying props that are just applied to the root element as is more often lately.  Is there a reason to not take advantage of vue's default behavior that you see here ?